src/utils.js

```diff
@@ -6,6 +6,9 @@ function getSafeUnicode(c) {
 }
 
 export const baseApiUrl = (subdomain, domain = DEFAULT_API_DOMAIN, domainDelimiter = '.') => {
+  if (domain === DEFAULT_DOMAIN || !domainDelimiter) {
```

Comment:
Nit but I think we can just do `if (!domainDelimiter) {` rather than also checking for the DEFAULT_DOMAIN

Reply:
Yeah, that's true. We can just check `domainDelimiter` then